core/scripts/upgrade-deps.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+##
+# @license Copyright 2025 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+##
+
+DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+LH_ROOT="$DIRNAME/../.."
+cd $LH_ROOT
+
+set -ex
+
+yarn upgrade --latest \
+    axe-core \
```

Comment:
Most of these are good to just pull the latest and then fix anything that breaks.

However, I think we should be a little more careful with axe-core. We should be making note of any new/deprecated axe checks with each new axe version.

Probably not a big deal though. Could you just add a note to `releasing.md` to check for new/deprecated axe checks after running this script?